pom.xml

```diff
@@ -1,7 +1,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>fr.bnf.digitools</groupId>
-  <artifactId>scannedPdf</artifactId>
+  <groupId>au.gov.nla</groupId>
```

Comment:
Should I just leave the group as is, since I'm not changing the package in the class files?